src/coreclr/jit/lowerriscv64.cpp

```diff
@@ -276,7 +276,8 @@ void Lowering::LowerBlockStore(GenTreeBlk* blkNode)
         }
         else
         {
-            blkNode->gtBlkOpKind = GenTreeBlk::BlkOpKindHelper;
+            blkNode->gtBlkOpKind =
+                blkNode->IsZeroingGcPointersOnHeap() ? GenTreeBlk::BlkOpKindLoop : GenTreeBlk::BlkOpKindHelper;
```

Comment:
It looks like the codegenriscv64.cpp code is unfinished (with "TODO"). Should this change be commented out (left as it was before), and open an issue to fix the codegen and enable it?